extensions/ipynb/src/helpers.ts

```diff
@@ -369,7 +374,7 @@ function convertStreamOutput(output: NotebookCellOutput): JupyterOutput {
 	const outputs = output.items
 		.filter((opit) => opit.mime === CellOutputMimeTypes.stderr || opit.mime === CellOutputMimeTypes.stdout)
 		.map((opit) => convertOutputMimeToJupyterOutput(opit.mime, opit.data as Uint8Array) as string)
-		.reduceRight<string[]>((prev, curr) => (Array.isArray(curr) ? prev.concat(...curr) : prev.concat(curr)), []);
+		.reduceRight<string[]>((prev, curr) => prev.concat(curr), []);
```

Comment:
Fixes the issue here https://github.com/microsoft/vscode/issues/129212

Reply:
I'll recommend we stop using `concat` completely. Filed separate issue and suggestions of improving the performance and  memory usage of `convertStreamOutput` https://github.com/microsoft/vscode/issues/129370#issue-952511326